packages/asset-value/src/types.ts

```diff
@@ -0,0 +1,38 @@
+import { Asset } from '@shapeshiftoss/asset-service'
+import { AssetId } from '@shapeshiftoss/caip'
+
+export enum AssetValueFormat {
+  BASE_UNIT = 1,
+  PRECISION = 2,
+}
+
+export type AssetValueParams = {
```

Comment:
personally would just add these types along side asset value class for easier readability in such a small package (could just reduce to single index.ts really imo)